src/title.cpp

```diff
@@ -0,0 +1,138 @@
+#include "title.h"
+#include "audio.h"
+#include "interop.hpp"
+#include "openre.h"
+#include "re2.h"
+
+using namespace openre::audio;
+
+namespace openre::title
+{
+    using Action = void (*)();
+
+    enum
+    {
+        TITLE_STATE_CAPCOM_LOGO,
+        TITLE_STATE_TITLE_LOAD,
+        TITLE_STATE_TITLE_MAIN,
+        TITLE_STATE_4TH_SURVIVOR_LOAD,
+        TITLE_STATE_4TH_SURVIVOR_MAIN,
+        TITLE_STATE_EXTREME_BATTLE_LOAD,
+        TITLE_STATE_EXTREME_BATTLE_MAIN,
+        TITLE_STATE_7,
+    };
+
+    static Action title_mv[] = {
+        (Action)0x00503680, // capcom_logo
+        (Action)0x005038B0, // title_init
+        (Action)0x00503A20, // title_main
+        (Action)0x00505460, // title_survivor_load
+        (Action)0x00505670, // title_survivor_main
+        (Action)0x00506F90, // title_extreme_load
+        (Action)0x00507100, // title_extreme_main
+        (Action)0x00507AB0  // ?
+    };
+
+    // 0x004D1150
+    static void pad_rep_set(uint32_t arg0, uint16_t arg1)
+    {
+        gGameTable.dword_98F074 = arg0;
+        gGameTable.word_98F078 = arg1;
+    }
+
+    // 0x00440480
+    static void add_sprt_v(int x, int y, int w, int h, int u, int v, int clut, int page, int depth, int isBack)
```

Comment:
This returns int

Reply:
Is it necessary though? The function is only called for the title I think and the result is not used.

Reply:
You're right! It might not be necessary